src/modules/launcher/PowerLauncher/Themes/Base.xaml

```diff
@@ -85,7 +85,7 @@
                                         Margin="{TemplateBinding Padding}"
                                         Grid.Row="0" />
 
-            <!--Scrollbar in thr rigth of ScrollViewer-->
+            <!--Scrollbar in the right of ScrollViewer-->
```

Comment:
Notes:
* My original pass didn't fix `thr` -- it's possible that I decided `thr` was a valid abbreviation for `thread`
* `in` might be better as `on`, or `to`, or something else...